### PR TITLE
[Admin] TerritorialCouncilMembershipLog add resolve action button

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

**Comment by illusionOfParadise** [comment]
[ch12050]

**Comment by illusionOfParadise** [comment]
[ch12050]

File: src/DataFixtures/ORM/LoadTerritorialCouncilMembershipLogData.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    public function load(ObjectManager $manager)
+    {
+        $membershipLog1 = new TerritorialCouncilMembershipLog(
+            TerritorialCouncilMembershipLog::TYPE_WARNING,
+            'Plusieurs conseils territorials ont été trouvés pour cette qualité',
+            $this->getReference('adherent-7'),
+            TerritorialCouncilQualityEnum::COMMITTEE_SUPERVISOR,
+            null,
+            [],
+            ['Seine-et-Marne (77), Essonne (91)']
```
Comment (kevin-verschaeve) [inline src/DataFixtures/ORM/LoadTerritorialCouncilMembershipLogData.php:21]: should this be :
```suggestion
            ['Seine-et-Marne (77)', 'Essonne (91)']
```
?

File: translations/messages.fr.yml [hunk 2]
```diff
@@ -909,6 +909,12 @@ territorial_council.membership.qualities.elected_candidate_adherent: Adhérents
 territorial_council.members_count: '1 membre|%count% membres'
 territorial_council.adherent_has_no_membership: L'adhérent ({{ email }}) n'est pas un membre du Conseil territorial "{{ territorialCouncil }}"
 
+#
+# TerritorialCouncilMembershipLog
+#
+territorial_council_membership_log.is_resolved: Ce log est déjà résolue.
+territorial_council_membership_log.is_not_resolved: Ce log n'est pas encore résolue.
```
Comment (kevin-verschaeve) [inline translations/messages.fr.yml:916]:
```suggestion
territorial_council_membership_log.is_resolved: Ce log est déjà résolu.
territorial_council_membership_log.is_not_resolved: Ce log n'est pas encore résolu.
```

**Review by ottaviano — APPROVED** [review]

**Pull request merged**